build-wheels.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e -u -x
+
+# Adapted from: https://github.com/pypa/python-manylinux-demo/blob/master/travis/build-wheels.sh
+
+PLAT=manylinux2014_x86_64
+
+function repair_wheel {
+    wheel="$1"
+    if ! auditwheel show "$wheel"; then
+        echo "Skipping non-platform wheel $wheel"
+    else
+        auditwheel repair "$wheel" --plat "$PLAT" -w /opt/genn/dist/
+    fi
+}
+
+
+# # Install a system package required by our library
+# yum install -y atlas-devel
+
+# Compile wheels
+for PYBIN in /opt/python/*/bin; do
+    # Exclude Python 3.12
+    if [[ "$PYBIN" == *"cp312"* ]]; then
+        continue
+    fi
+    # "${PYBIN}/pip" install -r /io/dev-requirements.txt
+    "${PYBIN}/pip" install numpy swig
+    # "${PYBIN}/pip" wheel /opt/genn/ --no-deps -w dist/
+    make install -j `lscpu -p | egrep -v '^#' | sort -u -t, -k 2,4 | wc -l`
+    make DYNAMIC=1 LIBRARY_DIRECTORY=${GENN_PATH}/pygenn/genn_wrapper/ -j `lscpu -p | egrep -v '^#' | sort -u -t, -k 2,4 | wc -l`
```

Comment:
this doesn't link against python so could be done once outside of the loop

Reply:
I must have misunderstood something then. The packages need to be installed within the loop still though don't they? Since GeNN uses them for bindings, doesn't that also mean it should be built within the loop?

Reply:
the stuff you're installing with pip definitely need to be done within the loop but ``make DYNAMICS=1 etc etc`` isn't Python-specific, it just builds GeNN as shared libraries and sticks them where the python build system expects them to be so can move it outside the loop (it probably won't do anything after the first iteration anyway cos nothing will have changed)